recipes/veque/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,16 @@
+
+#include <iostream>
+#include <string>
```

Comment:
```suggestion
```

Reply:
There are some `std::to_string` inside the `veque.hpp`

Reply:
But it should come from veque.hpp, correct?

Reply:
Yes, `veque.hpp` is missing that header

Reply:
Therefore, it may be better to patch it, it's clearly a compilation issue.

Reply:
Author here.  Thanks for catching and reporting this oversight.  It has been remedied in the latest release.